code/modules/recycling/disposal.dm

```diff
@@ -528,14 +528,23 @@
 
 /obj/machinery/disposal/hitby(atom/movable/AM)
 	. = ..()
-	if(istype(AM, /obj/item) && !istype(AM, /obj/item/projectile))
+	//CHOMPEdit: disposal dunking and autoflushes with mobs~ - Reo
```

Comment:
Only thing I see is this comment as leftover. Autoflushes are not in, so it shouldn't be mentioned that it does it.

Reword it to maybe: With the option to enable autoflush if desired

Reply:
Changed the comment to reflect code changes. Honestly yeah I completely forgot this one.